src/ControlTasks/control_task_base.py

```diff
@@ -0,0 +1,27 @@
+from abc import ABC, abstractmethod
+from typing import Dict
+from ..sfr import StateFieldRegistry
+
+""" Subclasses MUST implement the functions contained here """
+class ControlTaskBase(ABC):
```

Comment:
I think it's worth delineating what this class does. Is it just anything that is a process needs to use this class?